packages/react-test-renderer/shallow.js

```diff
@@ -9,7 +9,7 @@
 
 'use strict';
 
-const ReactShallowRenderer = require('./src/ReactShallowRenderer');
+const ReactShallowRenderer = require('react-shallow-renderer');
```

Comment:
Not sure if I should add a deprecation warning here (so that it would log whenever this module is imported).

Reply:
Related to this - not sure if I should've kept `ReactShallowRenderer.js` file and re-exported from there (perhaps, some dependency on tooling).

Reply:
This is fine but can you see if converting it to an ES module works: https://github.com/facebook/react/pull/18145/files#diff-d9693da2f86ad47019c4f8b94d7e5c3d

I believe this would cause the tests to fail so they need to be updated in the same way as I did in that PR.

Reply:
Seems like it worked and Rollup also did its thing 🙂